CICD/ExtensionMethods.cs

```diff
@@ -371,8 +370,8 @@ public static string GetVersion(this Project project)
 
         if (string.IsNullOrEmpty(version))
         {
-            // TODO: Create custom exception name MissingVersionException 
-            // TODO: In the exception, explain how to set the version 
+            // TODO: Create custom exception name MissingVersionException
+            // TODO: In the exception, explain how to set the version
```

Comment:
Do you need these TODO's?
```suggestion
            // TODO: Create custom exception name MissingVersionException
            // TODO: In the exception, explain how to set the version
```